service/builder/pipelines_builder.go

```diff
@@ -243,6 +261,25 @@ func (pb *PipelinesBuilder) buildFanoutExportersMetricsConsumer(exporterNames []
 	return processor.CreateMetricsFanOutConnector(exporters)
 }
 
+func (pb *PipelinesBuilder) buildFanoutExportersLogConsumer(
+	exporterNames []string,
+) consumer.LogConsumer {
+	builtExporters := pb.getBuiltExportersByNames(exporterNames)
+
+	// Optimize for the case when there is only one exporter, no need to create junction point.
+	if len(builtExporters) == 1 {
+		return builtExporters[0].le
+	}
+
+	var exporters []consumer.LogConsumer
+	for _, builtExp := range builtExporters {
+		exporters = append(exporters, builtExp.le)
+	}
```

Comment:
I know that it does not matter, but I like to write optimal code even if one time only:
```golang
exporters := make([]consumer.LogConsumer, len(builtExporters))
for _, builtExp := range builtExporters {
	exporters = append(exporters, builtExp.le)
}
```

Reply:
Done.